frontend/src/components/dashboard/ProfileBanners.tsx

```diff
@@ -67,6 +71,16 @@ export const ProfileBanners = (props: Props) => {
     );
   }
 
+  if (
+    isBundleAvailableInCountry(props.runtimeData) &&
+    isFlagActive(props.runtimeData, "bundle") &&
+    !props.profile.has_vpn
```

Comment:
There's two things I'd double check with Tony here. First: do we also want to show this banner if the user already has phone masking? Or even regular Premium?

The second thing is that a free user can have a bit of a banner overload; this is a possible state:

![image](https://user-images.githubusercontent.com/4251/198258959-7bb00e4d-0f9f-4df4-8f27-207261c9061a.png)

We might want to choose between showing either the `<LoyalistPremiumBanner>` _or_ the `<BundlePromoBanner>` in some cases? Specifically, if the user has one or more masks, but does not have Premium nor VPN yet, which banner do we show?

Reply:
Confirmed with Tony this behaviour is fine for now.